docs/sources/tutorials/stack_guide_graphite.md

```diff
@@ -169,7 +169,7 @@ Graphite api is a light weight version of graphite-web with only the API compone
 to install.
 
 ```
-pip install gunicorn graphite-api
+pip install --no-cache-dir gunicorn graphite-api
```

Comment:
```suggestion
pip install gunicorn graphite-api
```

This is docs meant for end users. I'd rather avoid adding extra arguments and increasing the complexity, especially since the cache is presumably there for a reason when using pip on a regular operating system and not within the context of a Docker container.

Reply:
Agree with @sakjur.